docs/faq/client.md

```diff
@@ -0,0 +1,14 @@
+# FAQ: LeoFS Clients
+
+<!-- toc -->
+
+## Name Resolution
+
+* There are two ways to access LeoGateway from LeoFS Clients.
```

Comment:
It is ideal to fix this line as below:


There are two ways to access LeoGateway from LeoFS' clients.